kolibri/plugins/device_management/assets/src/views/SelectContentPage/index.vue

```diff
@@ -41,10 +41,7 @@
           v-if="transferredChannel && onDeviceInfoIsReady"
           class="updates"
         >
-          <div
-            v-if="newVersionAvailable"
-            class="updates-available"
-          >
+          <div v-if="newVersionAvailable">
```

Comment:
some tests might use this class as a selector

Reply:
udpated